jarvis_files/launch_scripts/ssh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+xdotool key --window $1 type "ssh -X mrover@10.1.0.1"
```

Comment:
might be an issue when using 2 jetsons

Reply:
Yeah. That is specifically designed to work with just one jetson. It would need to be modified to work with two